com.endlessnetwork.aqueducts.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "com.endlessnetwork.aqueducts",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "com.endlessnetwork.aqueducts.sh",
+  "tags": ["proprietary"],
+    "finish-args" : [
+        "--share=ipc", 
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=host:create"

```

Comment:
wait what? why are you asking to create a host filesystem? That should just be *there* if you need the ability to write to the host then a) just ask for that and b) I'm aware of the educational context but this needs to be much better scoped to the specific areas you need.

Reply:
Hrm.  Sorry about that, the studio included that bit I'm assuming because the work in progress version intends to have a level editor.  I'm just going to pull it out for now and when that feature appears in the future I'll make sure something more appropriately scoped is used.  Sorry again, I should have caught that.